types/prismic-dom/index.d.ts

```diff
@@ -18,3 +32,25 @@ export const Link: Link;
 
 declare const _default: { RichText: RichText, Link: Link };
 export default _default;
+
+export type Elements = {
```

Comment:
Are you definitely sure you can't re-export this from the `richtext` library:

https://github.com/prismicio/prismic-richtext/blob/master/src/types.ts#L3-L23

I think you'll need to do a `typeof NODE_TYPES` to get the correct shape of `NODE_TYPES`, but I am not 100% sure.

Reply:
```
...
../ramda/index.d.ts(2688,40): error TS2304: Cannot find name 'T'.
../ramda/index.d.ts(2688,46): error TS2304: Cannot find name 'U'.
../ramda/index.d.ts(2688,52): error TS2304: Cannot find name 'TResult'.
../ramda/index.d.ts(2688,60): error TS1005: ';' expected.
../ramda/index.d.ts(2688,84): error TS2304: Cannot find name 'T'.
../ramda/index.d.ts(2688,109): error TS2304: Cannot find name 'U'.
../ramda/index.d.ts(2688,116): error TS2304: Cannot find name 'TResult'.
../ramda/index.d.ts(2689,5): error TS1128: Declaration or statement expected.
../ramda/index.d.ts(2690,1): error TS1128: Declaration or statement expected.
```